features/support/form_helpers.rb

```diff
@@ -291,6 +291,14 @@ def pass_companies_house_number_validation
     options
   end
 
+  def pass_postcode_validation
+    options = {}
+    if find_elements_by_xpath("//input[@name='labAddressPostcode']").length > 0
```

Comment:
We could make this method a bit more versatile if we want, in case more postcode fields get validated in the future. It could look something like this: (I haven't tested this code)

```
def pass_postcode_validation
    options = {}
    postcode_elements = find_elements_by_xpath("//input[contains(local-name(), 'Postcode')]")
    if postcode_elements.length > 0
      element_name = postcode_elements[0]["name"]
      options[element_name] = "AB1 2CD"
    end
    options
  end
```

Reply:
It's interesting - we take a postcode in the company details page, but clearly the validation is more lax there...

Reply:
Yep, the API enforces a postcode <= 10 characters. The supplier-fe company details form only enforces <= 100. 🤔

Reply:
have made a tech debt ticket for this inconsistency. Will update this method as you suggest :)

Reply:
Cool :)

Reply:
I will be interested to see what comes out of that tech debt ticket :)